File: backend/ReadyBusinesses.Common/Dto/Recommendation/ExpertRecommendationDto.cs
using ReadyBusinesses.Common.Dto.Criteria;

namespace ReadyBusinesses.Common.Dto.Recommendation;

public class ExpertRecommendationDto
{
    public RecommendationDto Recommendation { get; set; } = null!;

    public string ExpertName { get; set; } = string.Empty;
    
    public double TotalScore { get; set; }

    public IEnumerable<CriteriaEstimateDto> CriteriaEstimates { get; set; } = [];
    
    public DateTime Date { get; set; }
}

File: frontend/src/Types/Recommendation/expertRecommendationDto.ts
import { RecommendationDto } from './recommendationDto.ts';
import { CriteriaEstimateDto } from '../Criteria/criteriaEstimateDto.ts';

export type ExpertRecommendationDto = {
  recommendation: RecommendationDto;
  expertName: string;
  date: Date;
  totalScore: number;
  criteriaEstimates: CriteriaEstimateDto[];
}